### PR TITLE
Metrics Webapp Review

### PR DESCRIPTION
**Pull request by akezeth**
### Metrics Webapp Review
- Used React documentation.
- Used React components.
- Used React props.
- Used React Router.
- Connected React and Redux.
- Handled events in a React app.
- Wrote integration tests with a React testing library.
- Used styles in a React app.
- Used React life cycle methods.
- Applied React best practices and language style guides in code.
- Used store, actions and reducers in React.

Presentation link: https://www.loom.com/share/ad7bde3231eb4614b51617f5de9cefb0

### CODE REVIEW

File: src/components/CryptoCurrency.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                  <div className="up">
+                    {changePercent24Hr < 0 ? (
+                      <span>
+                        <FaChevronDown color="red" />
+                        {Math.abs(changePercent24Hr).toFixed(2)}
+                      </span>
+                    ) : (
+                      <span>
+                        <FaChevronUp color="green" />
+                        {Math.abs(changePercent24Hr).toFixed(2)}
```
Comment (Munsa1) [inline src/components/CryptoCurrency.js:60]: - [ ] It is a requirement that we should follow the design guidelines including the :
- typography: font face, size, and weight
- Layout: composition and space between elements.

Kindly change the font-family to either lato or gill-sans

File: src/__tests__/CryptoCurrency.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import CryptoCurrency from '../components/CryptoCurrency';
+import '@testing-library/jest-dom/extend-expect';
+
+jest.mock('react-redux');
```
Comment (Munsa1) [inline src/__tests__/CryptoCurrency.test.js:7]: - [ ] Good job adding tests for the `CryptoCurrency` page, kindly do so for the `Details` page as well as this is a requirement to test several components of our app.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @akezeth, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>
## Highlight
Good Video Presentation ✔️ 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by akezeth** [comment]
- Changed the font-family to Lato.
- Added test for the Details component.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @akezeth 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Descriptive PR.
- Good commit messages.
- Correct git-flow.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**